app/models/barcode.rb

```diff
@@ -35,14 +35,15 @@ class Barcode < ApplicationRecord
     cgap_plate: 14,
     cgap_rack: 15,
     glasgow: 16,
-    cambridge_a_z: 17
+    cambridge_a_z: 17,
+    high_throughput_cherrypick: 18
   }
 
   # Barcode formats which may be submitted via sample manifests
   FOREIGN_BARCODE_FORMATS = %i[cgap fluidx_barcode fluidigm
                                uk_biocentre_v1 uk_biocentre_v2 uk_biocentre_unid
                                alderly_park_v1 alderly_park_v2 uk_biocentre_v3 cgap_plate cgap_rack
-                               glasgow cambridge_a_z].freeze
+                               glasgow cambridge_a_z high_throughput_cherrypick].freeze
```

Comment:
wonder if it might be better to call this heron_tailed as that is what HT stands for?

Reply:
Ahh yes... that makes more sense.